sdcm/mgmt.py

```diff
@@ -343,35 +343,36 @@ def get_hosts_health(self):
         """
         Gets the Manager's Cluster Nodes status
         """
-        # $ sctool status -c cluster1
-        # ╭───────────────┬────────┬──────────╮
-        # │ Host          │ Status │ RTT (ms) │
-        # ├───────────────┼────────┼──────────┤
-        # │ 18.234.77.216 │ UP     │ 0.92761  │
-        # │ 54.203.234.42 │ DOWN   │ 0        │
-        # ╰───────────────┴────────┴──────────╯
-        # or where all nodes up:
-        # ╭────────────────┬────────┬──────────╮
-        # │ Host           │ Status │ RTT (ms) │
-        # ├────────────────┼────────┼──────────┤
-        # │ 34.220.201.186 │ UP     │ 152.76   │
-        # │ 54.164.46.117  │ UP     │ 0.82     │
-        # ╰────────────────┴────────┴──────────╯
+        # Datacenter: us-eastscylla_node_east
+        # $ sctool status  -c mgr_cluster1
+        # ╭──────────┬────────────────╮
+        # │ CQL      │ Host           │
+        # ├──────────┼────────────────┤
+        # │ UP (1ms) │ 18.233.164.181 │
+        # ╰──────────┴────────────────╯
+        # Datacenter: us-west-2scylla_node_west
+        # ╭────────────┬───────────────╮
+        # │ CQL        │ Host          │
+        # ├────────────┼───────────────┤
+        # │ UP (180ms) │ 54.245.183.30 │
+        # ╰────────────┴───────────────╯
         cmd = "status -c {}".format(self.id)
-        parsed_table = self.sctool.run(cmd=cmd, is_verify_errorless_result=True)
+        dict_status_tables = self.sctool.run(cmd=cmd, is_verify_errorless_result=True, is_multiple_tables=True)
 
```

Comment:
is_multiple_tables means more than one DC?
if so, I don't think True should be the default...